### PR TITLE
feat(as-3238): integration of pipeline and structural refactor of azure functions

### PR DESCRIPTION
**Pull request by demzzydev**
integration of pipeline and structural refactor of azure functions

## Ticket Number
AS-3238
https://pins-ds.atlassian.net/browse/AS-3238

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/horizon-functions/azure-functions-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Docker
+# Build and push an image to Azure Container Registry
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+  branches:
+    include:
+      - feature/3074b-appeal-service-api-pipeline-tests
+  #    - master
```
Comment (swdpcomputing) [inline packages/horizon-functions/azure-functions-pipeline.yml:9]: Describe comment-outs please

**Review by swdpcomputing — APPROVED** [review]
LGTM

File: packages/horizon-functions/azure-functions-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Docker
+# Build and push an image to Azure Container Registry
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+  branches:
+    include:
+      - feature/3074b-appeal-service-api-pipeline-tests
```
Comment (rcmsoftwaredevelopment) [inline packages/horizon-functions/azure-functions-pipeline.yml:8]: Is this branch still valid?

**Review by rcmsoftwaredevelopment — CHANGES_REQUESTED** [review]

**Comment by demzzydev** [comment]
Closing PR due to git issues

**Pull request closed without merging**